Navaids/FIXES_SIDS-STARS.txt

```diff
@@ -1,9 +1,5 @@
-05ADM N053.27.22.520 W001.00.51.400
 05DME N053.29.39.390 W000.59.37.900
 15DMA N053.30.39.210 W000.59.42.990
-15DME N053.26.29.140 W001.01.37.830
-25DME N053.25.23.370 W001.01.17.360
```

Comment:
We need to keep 25DME as it's in the procedures we kept

Reply:
Fixed